sources/packages/forms/src/form-definitions/studentprofilesummary.json

```diff
@@ -132,6 +132,8 @@
                             "components": [
                                 {
                                     "label": "First Name",
+                                    "labelWidth": "",
```

Comment:
Is there a reason why this page is used as form and not Vue component?
![image](https://user-images.githubusercontent.com/54600590/197844148-a7fdcf09-c271-4689-9c0d-c92517bd26a6.png)

@guru-aot  @andrewsignori-aot @ann-aot  ? Is there something decided before which I don't remember?

Same part of institution is Vue component

Reply:
Yes, there was a decision. I would be 100% supporting moving it to Vuetify.
I also mentioned in the past that I do not see the matter of this page since we can directly render the Institution Profile here which is the same accessed from to top right "Edit" button.

Reply:
@guru-aot we can create a tech debt if we are going to change this to vuetify.